hugegraph-server/Dockerfile

```diff
@@ -50,17 +50,17 @@ RUN set -x \
 
 # 2. Init HugeGraph Sever
 RUN set -e \
-    && pwd && cd /hugegraph/ \
+    && pwd && cd /hugegraph-server/ \
     && sed -i "s/^restserver.url.*$/restserver.url=http:\/\/0.0.0.0:8080/g" ./conf/rest-server.properties
 
 # 3. Init docker script
-COPY hugegraph-dist/docker/scripts/remote-connect.groovy ./scripts
-COPY hugegraph-dist/docker/scripts/detect-storage.groovy ./scripts
-COPY hugegraph-dist/docker/docker-entrypoint.sh .
+COPY hugegraph-server/hugegraph-dist/docker/scripts/remote-connect.groovy ./scripts
+COPY hugegraph-server/hugegraph-dist/docker/scripts/detect-storage.groovy ./scripts
+COPY hugegraph-server/hugegraph-dist/docker/docker-entrypoint.sh .
 RUN chmod 755 ./docker-entrypoint.sh 
 
 EXPOSE 8080
-VOLUME /hugegraph
+VOLUME /hugegraph-server
 
 ENTRYPOINT ["/usr/bin/dumb-init", "--"]
 CMD ["./docker-entrypoint.sh"]
```

Comment:
also test `docker stop` & `docker start` here for server

Reply:
https://hugegraph.feishu.cn/wiki/ThvlwPUHNiCpwPkjurkcw4mBnpf
tested.
Stop and start the server the data did not change.
But if i change the server image to the new version, it stuck when loading:
![image](https://github.com/apache/incubator-hugegraph/assets/49650772/b9a43e85-7f95-461b-b0ab-7ab3cf5d9771)